openpathsampling/tests/test_shooting.py

```diff
@@ -232,3 +232,17 @@ def test_probability_ratio_modified_coordinates(self, new_coord, expected):
         mod_traj += mytraj[idx+1:]
         prob = self.sel.probability_ratio(frame, mytraj, mod_traj, mod_frame)
         assert prob == expected
+
+
+class TestDeprecations(object):
+    @pytest.mark.parametrize('selector_class', [ShootingPointSelector,
+                                                FirstFrameSelector,
+                                                FinalFrameSelector])
```

Comment:
Any reason not to just update these now (and make a quick subclass/instance of the base class if it isn't abstract) to test the deprecation call?

Reply:
Got it. Thanks for the clarification!